requirements-test.txt

```diff
@@ -7,5 +7,6 @@ pytest-xdist==1.31.0
 pytest==5.3.5
 redis==3.4.1
 requests==2.23.0
-sentry_sdk==0.14.3
+# sentry_sdk==0.14.3
+git+git://github.com/getsentry/sentry-python@f0ba04ef161c0174f40a545268e92bea5162f5ec#egg=sentry_sdk
```

Comment:
@rhcarvalho Just a heads up, I'm now incorporating the utility methods from https://github.com/getsentry/sentry-python/pull/793 into this PR for testing. 

I'll revert this change later.